Assets/Scripts/Game/Questing/Quest.cs

```diff
@@ -599,7 +592,7 @@ public DaggerfallDateTime GetCurrentLogMessageTime()
                     return log.dateTime;
             }
 
-            return questStartTime;
+            return QuestStartTime;
```

Comment:
Returning the property isn't needed there. This is probably a leftover from your previous changes.